R/get_child_mortality.R

```diff
@@ -0,0 +1,161 @@
+#' Get child mortality rates
+#' 
+#' @description
+#' This function computes and returns the child mortality rates developed by the 
+#' UN Inter-Agency Group for Child Mortality Estimation, which includes UNICEF, 
+#' WHO, the World Bank, and the UN DESA Population Division. This data is retrieved
+#' from the World Bank website, and terms of use are found here: https://www.worldbank.org/en/about/legal/terms-of-use-for-datasets.
+#' 
+#' @details 
+#' Neonatal mortality: the probability of dying between the first 28 days of life, expressed per 1000 live births.
+#' Infant mortality: the probability of dying between birth and exact age 5, expressed per 1000 live births.
+#' Under 5 mortality rate: the probability of dying between birth and exact age 5, expressed per 1000 live births.
+#' 
+#' Future additions:
+#' Memory use improvements. Check of years and ages inputs. Uses at least an extra 2.6 MB. 
+#' Spell checks. Url check - make sure its still there.
+#' 
+#' @param country all, iso3c code, or country name in English. If not available or 
+#' misspelled, will throw error. To do: add a list of all countries/regions that we 
```

Comment:
I wouldn't worry about detailing error catches in this documentation - the informative error message should fill the user in if they trigger any errors